dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java

```diff
@@ -232,8 +234,13 @@ private Field dbSaveUpdate(final Field throwAwayField) throws DotDataException {
       }
 
       // normalize our velocityvar
+      final List<String> takenFieldVars = fieldsAlreadyAdded.stream().map(Field::variable).collect(
+              Collectors.toList());
+      // let's add GraphQL reserved field names to the taken fields vars list
+      takenFieldVars.addAll(InterfaceType.RESERVED_GRAPHQL_FIELD_NAMES);
```

Comment:
maybe we should move this logic to its own Util, something like:
```
class GrahpQLUtil {
    public List<String> getReservedWord(){
         return InterfaceType.RESERVED_GRAPHQL_FIELD_NAMES;
   }
}
```

Reply:
Done